packages/wrangler/src/__tests__/dev.test.tsx

```diff
@@ -1666,6 +1666,27 @@ describe("wrangler dev", () => {
 				`[Error: Cannot use Experimental Assets and tail consumers in the same Worker. Tail Workers are not yet supported for Workers with assets.]`
 			);
 		});
+
+		it("should error if --experimental-assets and --remote are used together", async () => {
+			fs.openSync("public", "w");
```

Comment:
Out of curiosity why use this rather than `mkdir()`?

Reply:
no specific reason. that's what existing tests used, so I went along with that. I'll go ahead and switch

Reply:
Cool. I wasn't actually asking for it to change, I was just interested in why this approach, since I had not seen it before.

Reply:
oh yeah, abs! very valid question. always love a good "pushback" ;)